Makefile

```diff
@@ -17,7 +17,7 @@ all:
 .PHONY: install-test-deps
 install-test-deps:
 	opam install 'menhir>20211230' cinaps 'ppx_expect>=v0.14.0' 'sexplib0.v0.14.0' \
-		ocamlformat.$$(awk -F = '$$1 == "version" {print $$2}' .ocamlformat) ocamlformat-rpc
+		ocamlformat.$$(awk -F = '$$1 == "version" {print $$2}' .ocamlformat) ocamlformat-rpc bos
```

Comment:
still some bos left overs here

Reply:
Thanks, I was about the remove the remaining leftovers, should be ok now :) I also renamed the executable to `run_test_e2e`